### PR TITLE
Added Avatar.html and Avatar.css

### PR DESCRIPTION
**Pull request by ayush65**

### CODE REVIEW

**Review by NtshVrm — CHANGES_REQUESTED** [review]
Try using consistent units i.e either px/rem/vh

**Review by narharikale — APPROVED** [review]
code looks clean, good to go ✔

File: components/Avatar/Avatar.css [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    width: 10rem;
+    height: 10rem;
+    font-size: 3rem;
+}
+
+.avatar-space{
+    margin-left:25px;
+    margin-bottom: 30px;
+    margin-top: 15px;
+}
```
Comment (kotesh-arya) [inline components/Avatar/Avatar.css]: You coud have used short-hand property for margin
like this =>   margin : 15px   0px   30px     25px;
                                   top   right   bottom   left

**Review by kotesh-arya — COMMENTED** [review]
Just that one improvisation suggestion from my side 👍🏻

**Review by NtshVrm — APPROVED** [review]

File: components/Avatar/Avatar.css [hunk 2]
```diff
@@ -0,0 +1,135 @@
+@import url('https://fonts.googleapis.com/css2?family=Dongle&family=Open+Sans:ital@1&family=Source+Sans+Pro:ital,wght@0,700;1,400&display=swap');
+@import url("../../style.css");
+@import url("../../color-palette/color.css");
+
+*, body{
+    margin:0;
+    padding:0;
+}
+.tab-header{
```
Comment (HarshithaKonduru214) [inline components/Avatar/Avatar.css:9]: Try using rems for values greater than 10px.

**Review by HarshithaKonduru214 — APPROVED** [review]

File: components/Avatar/Avatar.css (hunk 1 above)
Comment (ayush65) [reply components/Avatar/Avatar.css]: thanks i will update it , thanks for reviewing

**Review by kotesh-arya — APPROVED** [review]
Looks good now

**Comment by Himanshu-Dhawale** [comment]
Looks good to me. Code is also clean

File: components/Avatar/Avatar.css (hunk 2 above)
Comment (ayush65) [reply components/Avatar/Avatar.css:9]: yea i have implemented it . Thanks for reviewing

**Pull request merged**